control/preimage/src/helpers.rs

```diff
@@ -7,10 +7,10 @@ use crate::Context;
 
 use crate::bridge_hub_runtime::{self, RuntimeCall as BridgeHubRuntimeCall};
 
-#[cfg(feature = "polkadot")]
-use crate::relay_runtime::runtime_types::xcm::v2::OriginKind;
 #[cfg(any(feature = "westend", feature = "paseo"))]
 use crate::relay_runtime::runtime_types::xcm::v3::OriginKind;
+#[cfg(feature = "polkadot")]
+use crate::relay_runtime::runtime_types::xcm::v3::OriginKind;
```

Comment:
```suggestion
use crate::relay_runtime::runtime_types::xcm::v3::OriginKind;
```

Since all environments now use the same `OriginKind`.